pkg/object/objects.go

```diff
@@ -239,6 +239,7 @@ func ParseK8sObjectsFromYAMLManifest(manifest string) (K8sObjects, error) {
 	var objects K8sObjects
 
 	for _, yaml := range yamls {
+		yaml = removeCommentedLines(yaml)
```

Comment:
the below blank check logic can be added to the new functionand also update the function name

Reply:
Done.